documentation/specs/Badge.md

```diff
@@ -65,7 +65,7 @@ type Badge = {
    * Any additional text to support the primary label.
    * Only applicable on text-only badges.
    */
-  supportText?: ReactNode;
+  secondaryLabel?: ReactNode;
```

Comment:
seeing `supportText` in use wasn't super clear to me so i updated it to `secondaryLabel`. let me know if this is better or if i should revert or if there are better ideas.

Reply:
i think `secondaryLabel` is good